fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/ElectricVehicleUnit.java

```diff
@@ -83,7 +83,7 @@ public BatteryData getBatteryState() {
 
     @Override
     public void sendChargingStartRequest(String chargingStationId) {
-        if (getVehicleData() != null && !getVehicleData().isStopped()) {
+        if (getVehicleData() != null && getVehicleData().isStopped()) {
```

Comment:
prior to this PR, this was actually wrong. But we could also consider just removing this if-clause